prisma/populate.ts

```diff
@@ -71,6 +71,7 @@ async function populateTeachers() {
     await prisma.teacher.upsert({
       create: {
         user_id: teacher.userData.id,
+        siape: teacher.userData.siape
```

Comment:
Mudando o siape lá no mock, aqui fica assim:

```suggestion
        siape: teacher.siape
```